src/Core/Billing/Queries/IGetSubscriptionQuery.cs

```diff
@@ -0,0 +1,14 @@
+﻿using Bit.Core.Entities;
+using Stripe;
+
+namespace Bit.Core.Billing.Queries;
+
+public interface IGetSubscriptionQuery
+{
+    /// <summary>
+    ///
```

Comment:
Summary missing

Reply:
@r-tome Whoops! Nice catch. I just added this - thanks.